lib/index.ts

```diff
@@ -26,6 +27,7 @@ async function download(
 
   try {
     await downloadUrl(url, local);
+    await plusx(local);
```

Comment:
What is this?

Reply:
chmod +x . downloaded files don't have `x` perm by default.

Reply:
My question - why is it needed if it wasn't there before? We're just switching the request library, right?

Reply:
Oh. It was actually there. I forgot to add this when I simplify functions in https://github.com/vercel/pkg-fetch/commit/e314198e538d55d0eafba15e064ca027e584d313 .